src/devices/Ssd1306/Command/SetMultiplexRatio.cs

```diff
@@ -13,7 +13,7 @@ public class SetMultiplexRatio : ICommand
         /// The output pads COM0-COM63 will be switched to the corresponding COM signal.
         /// </summary>
         /// <param name="multiplexRatio">Multiplex ratio with a range of 15-63.</param>
-        public SetMultiplexRatio(byte multiplexRatio = 0x63)
+        public SetMultiplexRatio(byte multiplexRatio = 63)
```

Comment:
There are a few variables that have defaults when the device is reset.  I thought it would be useful to have those as optional in the commands, but can remove if you think that approach is better.  I'm on the fence.

Reply:
If having it explicitly be part of the API surface is useful to the caller and we don't ever see it changing in the future, then it's fine to keep. The concern of exposing the value is that we can never change it in the future.

Reply:
I am not a domain expert here, so take my suggestion with a grain of salt. It could very well be fine to keep the default parameter. cc @joperezr - thoughts?